src/main/java/jp/ikedam/jenkins/plugins/updatesitesmanager/UpdateSitesManager.java

```diff
@@ -23,274 +23,205 @@
  */
 package jp.ikedam.jenkins.plugins.updatesitesmanager;
 
-import java.io.IOException;
-import java.io.UnsupportedEncodingException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-
-import javax.servlet.ServletException;
-
+import com.google.common.base.Function;
+import com.google.common.base.Predicate;
+import com.google.common.collect.Iterables;
+import hudson.Extension;
+import hudson.model.Descriptor.FormException;
+import hudson.model.ManagementLink;
+import hudson.model.UpdateSite;
 import jenkins.model.Jenkins;
-
-import net.sf.json.JSONObject;
-
+import jp.ikedam.jenkins.plugins.updatesitesmanager.internal.IgnoreNotPOST;
+import jp.ikedam.jenkins.plugins.updatesitesmanager.internal.OnlyAdminister;
+import jp.ikedam.jenkins.plugins.updatesitesmanager.internal.Sites;
 import org.apache.commons.lang.StringUtils;
-import org.kohsuke.stapler.ForwardToView;
-import org.kohsuke.stapler.HttpRedirect;
 import org.kohsuke.stapler.HttpResponse;
-import org.kohsuke.stapler.StaplerRequest;
-import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.HttpResponses;
 
-import hudson.Extension;
-import hudson.model.ManagementLink;
-import hudson.model.Descriptor;
-import hudson.model.Descriptor.FormException;
-import hudson.model.UpdateSite;
+import javax.annotation.Nullable;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.List;
+
+import static com.google.common.base.Predicates.not;
+import static com.google.common.collect.Lists.newArrayList;
+import static com.google.common.collect.Sets.newHashSet;
 
 /**
- * Pages for manage UpdateSites.
- * 
- * Provides following pages.
+ * Page for manage UpdateSites.
+ *
+ * Provides following page.
  * <ul>
- *   <li>Manage UpdateSites, shown in Manage Jenkins page</li>
- *   <li>Add New Site</li>
+ * <li>Manage UpdateSites, shown in Manage Jenkins page</li>
  * </ul>
- * 
- * And provides access to each updatesite configuration page.
  */
 @Extension(ordinal = Integer.MAX_VALUE - 410)   // show just after Manage Plugins (1.489 and later)
-public class UpdateSitesManager extends ManagementLink
-{
-    private Logger LOGGER = Logger.getLogger(UpdateSitesManager.class.getName());
-    
+public class UpdateSitesManager extends ManagementLink {
+
     public final static String URL = "updatesites";
-    
+
     /**
      * Return the name of the link shown in Manage Jenkins page.
-     * 
+     *
      * @return the name of the link.
      * @see hudson.model.Action#getDisplayName()
      */
     @Override
-    public String getDisplayName()
-    {
+    public String getDisplayName() {
         return Messages.UpdateSitesManager_DisplayName();
     }
-    
+

```

Comment:
I highly recommend you to disable auto formatter of your IDE (especially when contributing OSS).
Unrelated changes makes it hard for reviewers to see the actual change you want and to track changes with `git blame`.
(I don't think you need to fix them and remake this request)